assemble/bin/accumulo-cluster

```diff
@@ -177,16 +166,20 @@ function control_service() {
       # using the value of $host
       #
       if [[ $# -gt 3 ]]; then
-        debugAndRun ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "-a" "$host" "${@:4}"
+        debug "ACCUMULO_SERVICE_INSTANCE=\"$ACCUMULO_SERVICE_INSTANCE\" \"$bin/accumulo-service\" \"$service\" \"$control_cmd\" -a \"$host\" \"${*:4}\"" ||
+          ACCUMULO_SERVICE_INSTANCE="$ACCUMULO_SERVICE_INSTANCE" "$bin/accumulo-service" "$service" "$control_cmd" -a "$host" "${@:4}"
       else
-        debugAndRun ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "-a" "$host"
+        debug "ACCUMULO_SERVICE_INSTANCE=\"$ACCUMULO_SERVICE_INSTANCE\" \"$bin/accumulo-service\" \"$service\" \"$control_cmd\" -a \"$host\"" ||
+          ACCUMULO_SERVICE_INSTANCE="$ACCUMULO_SERVICE_INSTANCE" "$bin/accumulo-service" "$service" "$control_cmd" -a "$host"
       fi
     else
       if [[ $# -gt 3 ]]; then
         EXTRA_ARGS="${*:4}"
-        debugAndRun "$SSH" "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-a\" \"$host\" $EXTRA_ARGS '"
+        debug "$SSH \"$host\" \"bash -c 'ACCUMULO_SERVICE_INSTANCE=\\\"$ACCUMULO_SERVICE_INSTANCE\\\" \\\"$bin/accumulo-service\\\" \\\"$service\\\" \\\"$control_cmd\\\" -a \\\"$host\\\" $EXTRA_ARGS '\"" ||
```

Comment:
@ctubbsii - I don't remember all of this escaping when you and I were testing this. IIRC when we were testing, there was a line that performed an `echo` and then the following line that performed the command. I realize that the `echo` didn't print exactly what was executed, but it seemed better than all of this escaping.

Reply:
Right, I added that back in because I thought it would be beneficial to be able to see a copy-and-paste safe version of the command. But, we certainly don't have to have that... we just need to be mindful of the debugging output that it may require additional quoting and won't be able to be copy-and-pasted directly.

I can pull that part out, if you want. I just thought it was helpful, and wanted to demonstrate it here for consideration.